storage/localstore/gc.go

```diff
@@ -94,6 +94,13 @@ func (db *DB) collectGarbage() (collectedCount uint64, done bool, err error) {
 	db.batchMu.Lock()
 	defer db.batchMu.Unlock()
 
+	// run through the recently pinned chunks and
+	// remove them from the gcIndex before iterating through gcIndex
+	err = db.removeChunksInExcludeIndexFromGC()
+	if err != nil {
+		log.Error("localstore exclude pinned chunks", "err", err)
```

Comment:
the error is only logged here. don't you want to return it as it is done in other error handling cases below?

Reply:
Yeah. Fixed.